warp10/src/main/java/io/warp10/continuum/egress/EgressExecHandler.java

```diff
@@ -321,12 +449,23 @@ public void handle(String target, Request baseRequest, HttpServletRequest req, H
         times.add(end - nano);
       }
 
+      if (forcedMacro) {
+        stack.macroClose();
+      }
+
       //
       // Make sure stack is balanced
       //
 
       stack.checkBalanced();
 
+      if (maxtime > 0) {
+        stack.push(maxtime);
+        TIMEBOX.apply(stack);
```

Comment:
This will modify the error message. For instance a simple `FAIL` will result in:
```
TIMEBOX encountered an exception while executing macro (Exception at 'FAIL' in section '[TOP]' (FAIL failed.))
```

Reply:
Can you be more elaborate?

Reply:
For instance, if `egress.timebox` is not set and no special header is set, a script with `FAIL` will return
`Exception at '=>FAIL<=' in section [TOP] (FAIL failed.)`

In the case `egress.timebox` is set and no special header is set, the same script will return 
`TIMEBOX encountered an exception while executing macro (Exception at 'FAIL' in section '[TOP]' (FAIL failed.))`

It may not be obvious to the user the execution was TIMEBOXed and the message can be confusing.